src/components/forms/RangeInput/RangeInput.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import classnames from 'classnames'
+
+import { Label } from '../Label/Label'
+
+interface RangeInputProps {
+  id?: string
+  className?: string
+  label?: React.ReactNode
+  hint?: React.ReactNode
+  inputRef?:
+    | string
+    | ((instance: HTMLInputElement | null) => void)
+    | React.RefObject<HTMLInputElement>
+    | null
+    | undefined
+}
+
+export const RangeInput = (
+  props: RangeInputProps & React.InputHTMLAttributes<HTMLInputElement>
+): React.ReactElement => {
+  // Range defaults to min = 0, max = 100, step = 1, and value = (max/2) if not specified.
+  const {
+    id = 'range-slider',
+    name = 'range',
```

Comment:
this is my 🔥  opinion and is more of a general comment about form inputs, but IMO `id` and `name` are so fundamental to building HTML forms that I don't think we need to provide default values, and I also think they should be required in the prop type.

Reply:
That's great context, keep the opinions coming I don't have any of my own yet.